ogr/services/github.py

```diff
@@ -66,8 +97,27 @@ def get_branches(self) -> List[str]:
     def get_description(self) -> str:
         return self.github_repo.description
 
-    def get_fork(self) -> Optional[GitProject]:
-        raise NotImplementedError
+    def get_fork(self, create: bool = True) -> Optional[GitProject]:
```

Comment:
```suggestion
    def get_fork(self, create: bool = True) -> Optional[GithubProject]:
```

Reply:
so I ran it locally to get used to it; and it's installed also!